### PR TITLE
Fixed ItCoherenceTests.testCoherenceServerRollingRestart test class failing intermittently

### PR DESCRIPTION
**Pull request by hzhao-github**
Jenkins:

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/13676/
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/13674/

### CODE REVIEW

**Review by anpanigr — APPROVED** [review]

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/FileUtils.java [hunk 1]
```diff
@@ -302,6 +302,35 @@ public static void makeDirectories(String namespace,
     });
   }
 
+  /**
+   * Create a directory in a pod in specified namespace.
```
Comment (sankarpn) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/FileUtils.java:306]:
```suggestion
   * Create directories in a pod in the specified namespace.
```

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/FileUtils.java [hunk 2]
```diff
@@ -302,6 +302,35 @@ public static void makeDirectories(String namespace,
...
+   * @param namespace The Kubernetes namespace that the pod is in
+   * @param pod The name of the Kubernetes pod where the command is expected to run
+   * @param container The container in the Pod where the command is to be run. If no
+   *                         container name is provided than the first container in the Pod is used.
+   * @param redirectToStdout copy process output to stdout
+   * @param directoryToCreate namespace in which the pod exists
+   */
+  public static void deleteDirectories(String namespace,
```
Comment (sankarpn) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/FileUtils.java:314]: The intent of the api is to create directories not delete, please rename the  method

**Review by sankarpn — CHANGES_REQUESTED** [review]
minor changes

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/FileUtils.java (hunk 1 above)
Comment (hzhao-github) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/FileUtils.java]: changed

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/FileUtils.java (hunk 2 above)
Comment (hzhao-github) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/FileUtils.java:314]: it's a api to delete the dir. Comments were wrong :(

**Review by sankarpn — CHANGES_REQUESTED** [review]
minor comments

**Review by sankarpn — APPROVED** [review]
lgtm

**Pull request merged**